README.md

```diff
@@ -541,6 +542,45 @@ fn main() {
 : a,b,c
 ```
 
+#### :main
+
+Auto wrap whole block body in a "fn main" function call if none exists.
+
+Since this is very handy in most code snippets, so the default value is "yes".
+"no" if you don't want this feature(for example if you don't want regex search slow things down).
+
+You can also set a default value by:
+``` elisp
+;; By setq this default to `nil`, you'll have to explict set params to ":main yes" in each block
+(setq rustic-babel-auto-wrap-main nil)
+```
+
+```
+#+begin_src rust
+let x = vec![1, 2, 3].iter().map(|&x| x + 1).collect::<Vec<_>>();
+println!("{:?}", x);
+#+end_src
+
+#+results:
+: [2, 3, 4]
+```
+
+NOTE: the regex match would unable to /understand/ `fn main()` within a raw string.
+
+```
+#+begin_src rust :exports both :main yes
+// won't work
+let r = "
+fn main() {
+  let x = "within raw";
+}
+";
+
+// but this works
+printlnt!("fn main{{}}")
```

Comment:
I think that's not something anybody uses. Do you really think we need this ?

Reply:
Yes, this is useless...